Pod/Classes/SEGAdobeIntegration.m

```diff
@@ -400,7 +532,9 @@ - (void)trackHeartbeatEvents:(SEGTrackPayload *)payload
         if (!self.config) {
             return;
         }
-        self.mediaHeartbeat = [self.heartbeatFactory createWithDelegate:nil andConfig:self.config];
+
+        [self.playbackDelegate startDelegateWithInitialTime]; // We want to capture this initial time as soon as possible.
```

Comment:
// We want to capture this initial time as soon as possible.

This goes into what I mentioned in https://github.com/segment-integrations/analytics-ios-integration-adobe-analytics/pull/28#issuecomment-350926881 - the android integration creates the delegate and the `mediaHeartbeat` during "Video Playback Started" rather than on initialization.